### ISSUE TITLE
Error using cleanLineNoise (line 57)

### ISSUE THREAD

**Issue by Samuel-Ba** [VisLab/EEG-Clean-Tools#27]
Hello,

I am getting an error when I run PrepPipeline0.55.4
Here is a snip of the aftermath:
![Screenshot 2021-04-03 043830](https://user-images.githubusercontent.com/81857062/113476108-8080c680-9436-11eb-930b-b0722345a942.png)
![image](https://user-images.githubusercontent.com/81857062/113476231-61366900-9437-11eb-8e43-94bbc4a3ab2e.png)

The same errors were present when I had the cleanline plugin installed, except the source code path was directed at cleanline directly, rather than in the PrePipeline utilities.

**Comment by Samuel-Ba** [comment]
The issue has been resolved. Disabling parallel pool aloud prepPipeline to execute normally.

**Comment by datalw** [comment]
> 
> 
> The issue has been resolved. Disabling parallel pool aloud prepPipeline to execute normally.

I have a similar erorr, while using cleanLineNoise() as shown below. I don't quite understand the solution "Disabling parallel pool aloud prepPipeline" though... Sorry, I'm newbie to prep. Could you please explain a bit more, how to solve this issue? Thanks!

```
Error using testSignificantFrequencies (line 76)
Undefined function 'finv' for input arguments of type 'double'.

Error in fitSignificantFrequencies (line 33)
[Fval, A, f, sig] = testSignificantFrequencies(data, lineNoise);

Error in removeLinesMovingWindow (line 61)
            fitSignificantFrequencies(datawin, f0, lineNoise);

Error in cleanLineNoise (line 57)
parfor ch = chans
```

**Comment by VisLab** [comment]
I believe this issue is due to parfor not allowing chans to contain non-consecutive integers in 

parfor ch = chans

This will be corrected in version 0.56.0.
